src/Arcus.WebApi.Logging/RequestTrackingMiddleware.cs

```diff
@@ -210,7 +211,26 @@ private void LogRequest(string requestBody, string responseBody, HttpContext htt
             Dictionary<string, object> telemetryContext = CreateTelemetryContext(requestBody, responseBody, httpContext.Request.Headers, _logger);
             telemetryContext.Add("Body", "Request body is now available in 'RequestBody' dimension");
 
-            _logger.LogRequest(httpContext.Request, httpContext.Response, duration, telemetryContext);
+            var operationName = BuildOperationName(httpContext);
+
+            _logger.LogRequest(httpContext.Request, httpContext.Response, operationName, duration, telemetryContext);
+        }
+
+        private static string BuildOperationName(HttpContext httpContext)
+        {
+            string operationName = null;
+
+            if (httpContext.Features.Get<Microsoft.AspNetCore.Http.Features.IEndpointFeature>()?.Endpoint is RouteEndpoint re)
+            {
+                operationName = re.RoutePattern.RawText;
+            }
+
+            if (String.IsNullOrWhiteSpace(operationName))
+            {
+                operationName = httpContext.Request.Path;
+            }
+
+            return $"{httpContext.Request.Method} {operationName}";
```

Comment:
We can just leave it `null` then the `{method} {path}` will automatically be filled-out as `operationName`.

Reply:
Like [this](https://github.com/arcus-azure/arcus.webapi/pull/410/commits/d788fff2e340d45fc76383811480c9b979856f88) you mean ?  
I'm not so sure about it tbh.

Reply:
No, completely `return operationName;` where `operationName: null`. Behind the scenes in Observability, this will be filled-out automatically. https://github.com/arcus-azure/arcus.observability/blob/1cef318d2afb01543d65c655e0e32462b51708e7/src/Arcus.Observability.Telemetry.Core/Logging/RequestLogEntry.cs#L163